node/src/BaseClient.ts

```diff
@@ -1948,6 +1950,124 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Computes the intersection of sorted sets given by the specified `keys` and returns a list of intersecting elements.
+     * To get the scores as well, see `zinterWithScores`.
+     * To store the result in a key as a sorted set, see `zinterStore`.
+     *
+     * When in cluster mode, all keys in `keys` must map to the same hash slot.
+     * 
+     * See https://valkey.io/commands/zinter/ for more details.
+     * 
+     * @param keys - The keys of the sorted sets.
+     * @returns The resulting array of intersecting elements.
+     * 
+     * @example
+     * ```typescript
+     * await client.zadd("key1", {"member1": 10.5, "member2": 8.2})
+     * await client.zadd("key2", {"member1": 9.5})
+     * await client.zinter(["key1", "key2"]) // Output: ['member1']
```

Comment:
Please fix other examples too
```suggestion
     * await client.zadd("key1", {"member1": 10.5, "member2": 8.2});
     * await client.zadd("key2", {"member1": 9.5});
     * const result = await client.zinter(["key1", "key2"]);
     * console.log(result); // Output: ['member1']
```